GE_Requirements.mediawiki

```diff
@@ -2,24 +2,38 @@
 
 == Development Lifecycle Requirements ==
 
 * Direct commits or self-commits to the master branch are not allowed. All contributions to master MUST come in the form of a Pull Request.
 * Pull Requests MUST include not only code changes but also test changes and documentation changes.
-* Pull Requests MUST be public and open during a reasonable period of time so that the community can check it and contribute if necessary
+* Pull Requests MUST be public and open during a reasonable period of time so that the community can check them and contribute if necessary
 * Pull Requests need to be approved (using the standard Github function or a 'LGTM' comment) by all the reviewers involved, i.e. all people that gave their feedback on it.
-* Pull Request approval MUST be issued by a thrid party. Nobody can auto-merge their contributions.
+* Pull Request approval MUST be issued by a third party. Nobody can auto-merge their contributions.
 * Before merging an open Pull Request Travis status MUST be green. 
 
 
 == Publication Requirements ==
 
 * The SCM Tool MUST be [https://github.com Github.com]. 
-* Github MUST be used during the <b>whole development lifecycle</b>. See [[development.mediawiki#Development_Lifecycle|Development Lifecycle]] which describes a set of best practices for collaborative open source development.
-* See [[GE_Checklist.mediawiki|GE Publication Checklist]] for a list of tasks to be done in order to publish a new GE.
+* Github MUST be used during the <b>whole development lifecycle</b>. See [[development.mediawiki#Development_Lifecycle|Development Lifecycle Recommendations in FIWARE]]
+which describes a set of best practices for collaborative open source development.
+* See [[GE_Checklist.mediawiki|Checklist for publishing FIWARE Generic Enablers]] for a list of tasks to be done in order to publish a new GE.
+
+== Requirements regarding reference to FIWARE ==
+
+The README.md file of the GitHub repo, or several GitHub repos, associated to a FIWARE GE, '''MUST''' include an Introduction/About section at the beginning which includes:
+
+* the following paragraph clearly showing that the product has been labeled as a FIWARE GE and introducing FIWARE: 
+::'<name of the product> is a FIWARE Generic Enabler. Therefore, it  can be integrated as part of any platform “Powered by FIWARE”. FIWARE is a curated framework of open source platform components which can be assembled together with other third-party platform components to accelerate the development of Smart Solutions. You can find more info at the [https://deveopers.fiware.org FIWARE developers] website and the [https://fiware.org FIWARE] website.
+The complete list of FIWARE GEs and Incubated FIWARE GEs can be found at the [https://catalogue.fiware.org FIWARE Catalogue]'
+::
+* the [[project_badges.mediawiki|Project Badges]] which are stated as mandatory
+* what major.minor release of FIWARE each release of the product is compatible with
```

Comment:
From my point of view, this shouldn't be part of the README.md. Github already have a release tab that should be used to do that, in a per-release way, e.g:

![imagen](https://user-images.githubusercontent.com/1534240/45627585-930e9c80-ba92-11e8-8a1c-348d51df6af2.png)

Reply:
I'd tend to agree with @fgalan  here - the problem is with the phrase **each release**

The compatibility of a versioned release x.y.z  of product viz-a-viz other FIWARE components should be  found in the release note.

Maybe something like:

> * The release note for each versioned release should include a list of  the **latest** version of interoperable enablers it has been tested against.

(Obviously this requirement isn't on the `README`)


However it is still useful to be able to use the `README` to determine compatibility of the current release. So maybe we could  just get round this by adding a add a _deviations_ requirement as well:

The README **MUST**  include either of the following phrases 

> "The FIWARE release X.Y.Z. of this product is fully compatible with all FIWARE GE components released as part of FIWARE release X.Y.Z. No interoperable breaking changes have been made since then."
> 
> "The FIWARE release X.Y.Z. of this product is fully compatible with all FIWARE GE components released as part of FIWARE release X.Y.Z. The current build of this product includes breaking compatibility changes since FIWARE release X.Y.Z. as listed below
> * XXX
> * YYY"
>

Reply:
fixed https://github.com/Fiware/developmentGuidelines/pull/10/commits/142967a06afb193072c488e3cf90f8e0aa2651f9